stan/math/prim/mat/err/check_lower_triangular.hpp

```diff
@@ -32,9 +31,11 @@ namespace stan {
      */
     template <typename T_y>
     inline bool
-    check_lower_triangular(const char* function,
-                           const char* name,
-                           const Eigen::Matrix<T_y, Dynamic, Dynamic>& y) {
+    check_lower_triangular(
+      const char* function,
+      const char* name,
+      const Eigen::Matrix<T_y, Eigen::Dynamic, Eigen::Dynamic>& y
+    ) {
```

Comment:
If it doesn't fit, you can break under the T_y, and start the Eigen::Dynamic lined up under T_y on the next line.